sh/cmd.go

```diff
@@ -132,11 +132,14 @@ func run(env map[string]string, stdout, stderr io.Writer, cmd string, args ...st
 	c.Stderr = stderr
 	c.Stdout = stdout
 	c.Stdin = os.Stdin
-	log.Println("exec:", cmd, strings.Join(args, " "))
+
+	var quoted []string; for i:=0; i<len(args); i++ { 
```

Comment:
this doesn't look properly go-formatted.  Also, you can just do a range over args to get the indices.
```suggestion
	var quoted []string 
	for i := range args {
```

Reply:
yes I was just starting to learn go after making this PR :D